.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,77 @@
+# Release vFIXME
+
+## Checklist
+
+- [ ] [Create a new github milestone](https://github.com/DataDog/dd-trace-go/milestones/new) for the next version and
+  move any open PRs and issues that are in the current release milestone to the new one.
+
+- [ ] Draft [the release note](https://github.com/DataDog/dd-trace-go/releases/new) and ask the teams for its review on
+  the `#guild-dd-go` channel.
```

Comment:
Who are "the teams". I would be very explicit here as far as who you need approval from if there are certain required approvers. Or if anyone can LGTM, then I'd just say "get at least one person on the #guild-dd-go channel to review."

Reply:
Clarified in https://github.com/DataDog/dd-trace-go/pull/1284/commits/c7fcf9fd43ff54cd030376b442d0249b8a639224